packages/frontend/src/pages/Profile/name.ts

```diff
@@ -1,8 +1,28 @@
-export const toTitleCase = (str: string | undefined | null) => {
+import { ActorType } from 'bff-types-generated';
+
+export const getActorType = (actor: { actorType?: ActorType | null; actorId?: string | null }):
```

Comment:
I am a bit concerned that `toTitleCase` gives different version of a string if we can't consistently provide a correct type (company | person), not only because it's used for rendering, but some parts in the code for matching ids. 

I suggest reverting introducing type! Hopefully this will be fixed by backend, but at least you fixed an obvious incorrect rendering of a name, which is good!

Also, I think you should add _unit_ tests with @testing-library/react, cf. html.spec.tsx